.github/workflows/build.yml

```diff
@@ -0,0 +1,37 @@
+---
```

Comment:
We are in the process of building a new CI/CD pipeline. So most of the assets will most likely end up being deprecated